src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/mapper/conversation/ConversationsResourceMapper.kt

```diff
@@ -0,0 +1,13 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.resource.mapper.conversation
+
+import org.mapstruct.Mapper
+import org.mapstruct.Mapping
+import uk.gov.justice.digital.hmpps.domain.learningandworkprogress.conversation.dto.CreateConversationDto
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.resource.model.CreateConversationRequest
+
+@Mapper()
+interface ConversationsResourceMapper {
```

Comment:
MapStruct is pretty cool, and we use it a lot for these simple class mappers 👍 
Have you used it before / in other MoJ projects?

Reply:
No, first time, but seems to work really well. When I set this mapper up I assumed it wouldn't work first time and would require some debugging, but it worked first time and exactly as I was hoping.